### PR TITLE
Adds georeference analysis 

### PR DESCRIPTION
**Pull request by javierarce**
Adds georeference analysis.

screen shot 2016-07-04 at 14 42 36

### CODE REVIEW

**Comment by javierarce** [comment]
CR: @viddo can you have a look, please?

File: lib/assets/javascripts/cartodb3/editor/layers/layer-content-views/analyses/analysis-form-models/georeference-form-model.js [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+  /*
+  /**
+   * @override {BaseAnalysisFormModel._setSchema}
+   */
+  _setSchema: function () {
+    BaseAnalysisFormModel.prototype._setSchema.call(this, this._filterSchemaFieldsByType({
+      source: {
+        type: 'Select',
+        text: _t('editor.layers.analysis-form.source'),
+        options: [ this.get('source') ],
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/editor/layers/layer-content-views/analyses/analysis-form-models/georeference-form-model.js]: Please, start using the thing we defined here: https://github.com/CartoDB/cartodb/pull/8411

So we can show more info about the source node.

**Comment by xavijam** [comment]
What about the credits thing? Don't we need it?

**Comment by javierarce** [comment]
@xavijam: not right now, we are going to use Mapzen. @javisantana is on it.

**Comment by xavijam** [comment]
Ok, check the comment I made and ready to go ;). :es:

**Comment by viddo** [comment]
👍

**Comment by saleiva** [comment]
yay?

**Pull request merged**